### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by cannikin**
Live preview here: https://wip--toml-io.netlify.app/en/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# toml.io
+
+The public facing site at https://toml.io
+
+## Contributing
+
+* Copy spec/en to create new locale directory
```
Comment (pradyunsg) [inline README.md:7]:
```suggestion
* Copy `specs/en/` to create new locale directory
```

**Comment by cannikin** [comment]
You found that amazingly fast! 😅Those were just bullet point notes to myself, I'm going to do a better job of explaining soon, promise!

**Comment by pradyunsg** [comment]
@cannikin Yay! Super excited to see you working on this again! ^>^

Couple of questions:

- Dropping sections (specifically, the final 3) would be OK with this change, correct?
  - I'm thinking of moving the English specification out of the README, and wondering how that would interact with this.
- Can we drop the "1.0" from the logo in the navbar? I find it confusing since it shows up even when looking at 0.5.0 or others. Plus, it'll be another thing to update when we do a 1.1. ;)

File: code/html/spec.html.template [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      <div class="fixed bg-white z-20 left-0 right-0 mt-3 pl-4 py-4 border-b lg:hidden" data-target="nav.menuLink">
+        <div class="relative">
+          <a href="#" class="block font-semibold lg:hidden text-orange-800" data-action="click->nav#onMenuClick">Menu</a>
+          <div class="pointer-events-none absolute inset-y-0 right-0 flex items-center px-2 text-gray-700">
+            <svg class="fill-current h-4 w-4" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path d="M9.293 12.95l.707.707L15.657 8l-1.414-1.414L10 10.828 5.757 6.586 4.343 8z"></path></svg>
+          </div>
+        </div>
+      </div>
+
+      <nav class="hidden fixed w-full overflow-y-scroll top-0 bottom-0 bg-white z-10 py-32 pb-8 px-4 leading-snug lg:w-68 lg:block lg:pl-8 lg:py-24" data-target="nav.menu">
```
Comment (pradyunsg) [inline code/html/spec.html.template:17]: I feel like a bit more padding at the bottom would be nice, now that there's no footer. 🙈

**Comment by cannikin** [comment]
> Dropping sections (specifically, the final 3) would be OK with this change, correct?

Do you mean the last three sections on the spec itself? Comparison, Get Involved and Wiki? Those would be safe to remove yes!

> Can we drop the "1.0" from the logo in the navbar?

I thought that would be used to show what the current, latest official release is (I've got to update my build script to actually pull the latest version, instead of hardcoding "1.0" in there).

I'm going to tweak the "Version" and "Language" nav selectors so they actually show the version/language you're currently viewing. Do you think that would help with the ambiguity of that version next to the logo?

**Comment by pradyunsg** [comment]
> Do you think that would help with the ambiguity of that version next to the logo?

Maybe? Some part of me feels like this is over-engineering but I'm speaking with definitely-not-an-expert-at-this eyes here.

Plus, https://rachelbythebay.com/w/2013/06/05/duck/ comes to mind whenever I'm looking at a webpage that someone else is working on, so I'm very wary of being too nitpicky -- feel free to call me out on that.

**Comment by cannikin** [comment]
haha No problem! Check it out now and see how that feels. The Version and Language dropdowns show whatever you're currently viewing and the version next to the logo is always the latest official (english) spec version. Maybe I can even sneak in little "latest" label in there somehow...

**Comment by pradyunsg** [comment]
I still prefer not having any text beside the logo, but this works for me. :)

**Comment by pradyunsg** [comment]
@cannikin is there anything more to be done here? Can we merge this in and iterate off that?

I'm kinda itching to just retitle this PR to "initial implementation" and merge it. We don't have to get this perfect on the first try (perfect is the enemy of good enough) and it's probably better to have something merged in to iterate on together. :)

**Comment by cannikin** [comment]
Yep, I've been waiting to hear from everyone else about feedback, but I've been ready to go for years! 😃

Anyone know where the domain and DNS is? Does @mojombo still own that? We could deploy to Netlify and have it live there, or figure out some way to turn the static content generated by `yarn build` into GitHub Pages?

**Comment by pradyunsg** [comment]
This just needs to merge into master IIUC, and master was hooked up to toml.io (via Netlify) by @mojombo last I checked. :)

**Comment by cannikin** [comment]
In that case, let's get this thing live! MERGING

**Comment by cannikin** [comment]
https://toml.io !

**Comment by pradyunsg** [comment]
It's live: https://toml.io/en/ :)

**Comment by pradyunsg** [comment]
@cannikin I think you can also see the security advisories on this repository -- lemme know if you're updating the dependencies or would prefer that I do that. :)

**Comment by cannikin** [comment]
If you're okay to do it going forward that's fine by me! Thanks for taking care of those outstanding ones, I forgot all about them!

**Comment by pradyunsg** [comment]
Actually, could you do this initial round -- I should go to bed (it's 3am!). :)

**Comment by cannikin** [comment]
No problem, I'm on it! Thanks for the ones you already did! :)

**Comment by cannikin** [comment]
@pradyunsg Okay I updated all dependencies and cleared all the existing dependabot alerts, but now I've got two new ones instead. 😒 Since those are dependancies of dependancies is my only hope that I add those directly to `package.json` and hope they satisfy the requirements of the other packages that need them? And if not, is there any hope for satisfying those warnings? Other than just dismissing them? haha

**Comment by pradyunsg** [comment]
(narrator voice) he didn't sleep.

These two look like RCE issues -- which isn't a big deal for us, since they're used during development or deployment off master, and the website is a static website anyway. I say it's fine to ignore them for now. We can come back to them later, if we find them annoying. :)

**Pull request merged**